File: s10_Service/DownloadService/app/src/main/java/com/omeryildizce/downloadservice/MainActivity.java
package com.omeryildizce.downloadservice;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
    }

    public void download(View view) {
        Intent intent = new Intent(getApplicationContext(), DownloadService.class);
        startService(intent);
    }

    public void stop(View view) {
        Intent intent = new Intent(getApplicationContext(), DownloadService.class);
        stopService(intent);
    }
}

File: s01_Navigation/Jetpack Navigation/app/src/main/java/com/example/jetpacknavigation/MainActivity.kt
package com.example.jetpacknavigation

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import androidx.navigation.NavController
import androidx.navigation.Navigation
import androidx.navigation.ui.NavigationUI
import com.example.jetpacknavigation.databinding.ActivityMainBinding

class MainActivity : AppCompatActivity() {
    private lateinit var binding:ActivityMainBinding
    private lateinit var navigationControler :NavController

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        binding = ActivityMainBinding.inflate(layoutInflater)
        setContentView(binding.root)


    }


}